src/DataGenerator/Doctrine/Repository/TaxonRepository.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusVueStorefront2Plugin\DataGenerator\Doctrine\Repository;
+
+use Sylius\Bundle\TaxonomyBundle\Doctrine\ORM\TaxonRepository as BaseTaxonRepository;
+use Sylius\Component\Core\Model\TaxonInterface;
+
+final class TaxonRepository extends BaseTaxonRepository implements TaxonRepositoryInterface
```

Comment:
The queries from repository should be tested with PHPUnit integration tests. This applies for all repository classes.